.travis.yml

```diff
@@ -65,6 +65,13 @@ install:
   - pip install -r requirements.txt
   - pip install -r docs/requirements.txt
   - pip install -e .
+  - |
+    echo "\
+
+    [utils.iers.iers]
+    iers_auto_url = https://storage.googleapis.com/panoptes-resources/iers/ser7.dat
+    iers_auto_url_mirror = https://storage.googleapis.com/panoptes-resources/iers/ser7.dat
+    " >> $HOME/.astropy/config/astropy.cfg
```

Comment:
We need to do this both via config file and in `pocs/utils/data.py` because astroplan still isn't properly using the config file, or some other reason?

Reply:
For some reason it doesn't seem to be working properly on travis so had to duplicate here rather than just using the hard-coded entry in `data.py` itself (which we can and should move to our config system).